README.md

```diff
@@ -77,21 +77,21 @@ You can access the database through the Adminer front-end or using a local Postg
 5. Create a superuser so that you can log into `http://localhost:8000/admin` by running the following in your terminal:
 
 ```bash
-$ docker-compose run --rm app ./manage.py createsuperuser
+$ docker-compose run --rm app python ./manage.py createsuperuser
```

Comment:
Whoa, I thought these `python` changes were merged in already from https://github.com/codebuddies/backend/pull/147. Wonder why they appeared in your branch again.

Reply:
Solved by rebasing